### ISSUE TITLE
Chrome icon/logo showing in Settings

### ISSUE THREAD

**Issue by Saoiray** [brave/brave-browser#33099]
## Description 

Chrome icon/logo shows in settings while on dark mode.

## Steps to Reproduce

   1. Open Settings
   2. Make sure you're in Dark mode
   3. Make window small enough that there's a hamburger menu on the left.
   4. Click the hamburger menu and you'll see the Chrome icon on the top. 

## Actual result:

![image](https://github.com/brave/brave-browser/assets/99627477/2205e05f-87b1-47bb-8762-4ddb87097567)

## Expected result:
Should have Brave icon/logo as seen on Light mode. 
![image](https://github.com/brave/brave-browser/assets/99627477/8f9279f1-b90c-4bb3-bcb3-d4d51e43ec1c)

## Brave version (brave://version info)
Release: 1.58.27
Nightly: 1.60.30 and 1.60.33

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel?  Didn't test
- Can you reproduce this issue with the nightly channel? Yes

**Comment by diracdeltas** [comment]
can repro. cc @petemill @rebron

**Comment by stephendonner** [comment]
Also reproduces in 

```
Brave | 1.60.34 Chromium: 117.0.5938.88 (Official Build) nightly (x86_64)
-- | --
Revision | 780ad58fcc8bc6a420bb0b5502669ccef653a907
OS | macOS Version 11.7.10 (Build 20G1427)
```

<img width="731" alt="Screen Shot 2023-09-18 at 1 47 53 PM" src="https://github.com/brave/brave-browser/assets/387249/423474f1-fb42-45d2-add3-fb376e349704">

**Comment by GeetaSarvadnya** [comment]
Dup of #https://github.com/brave/brave-browser/issues/31355

**Comment by GeetaSarvadnya** [comment]
@simonhong I have closed this issue on behalf of #31355

**Comment by Saoiray** [comment]
> Dup of #31355

Bah, I tried looking through but missed it. Thanks for sharing and closing it up. Sorry had made a dupe.
